go/test/endtoend/tabletmanager/main_test.go

```diff
@@ -185,6 +185,11 @@ func tmcPrimaryPosition(ctx context.Context, tabletGrpcPort int) (string, error)
 	return tmClient.PrimaryPosition(ctx, vtablet)
 }
 
+func tmcGetGlobalStatusVars(ctx context.Context, tabletGrpcPort int, statuses []string) (map[string]string, error) {
```

Comment:
Really nitty, but IMO `varNames` is better than `statuses` throughout.

Reply:
or alternatively `variables`.

Reply:
Changed to `variables`.